hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/RootProxyWorldUpdater.java

```diff
@@ -46,7 +46,7 @@ public class RootProxyWorldUpdater extends ProxyWorldUpdater {
 
     private boolean committed = false;
     private List<ContractID> createdContractIds;
-    private List<ContractNonceInfo> updatedContractNonces;
+    private List<ContractNonceInfo> updatedContractNonces = new ArrayList<>();
```

Comment:
Could initialize to the `Collections.emptyList()` singleton here to avoid an extra object creation.

Reply:
Yes, thats true. Done. :)